src/service/translate.js

```diff
@@ -709,6 +710,29 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
 
   /**
    * @ngdoc function
+   * @name pascalprecht.translate.$translateProvider#useForceAsyncReload
+   * @methodOf pascalprecht.translate.$translateProvider
+   *
+   * @description
+   * If force async reload is enabled, async loader will be always called
+   * even if $translationTable already contains the language key.
+   *
+   * Example:
+   * <pre>
+   *  app.config(function ($translateProvider) {
+   *    $translateProvider.useForceAsyncReload(true);
+   *  });
+   * </pre>
+   *
+   * @param {boolean} boolean
+   */
+  this.useForceAsyncReload = function (value) {
+    $forceAsyncReloadEnabled = !(!value);
```

Comment:
I do not know what it does, I just copied it from the code snippet of `$translate.usePostCompiling` :sweat_smile: